src/components/pages/PendingApplications/ApplicationModal.less

```diff
@@ -0,0 +1,74 @@
+.modalStyleMentee,
+.modalStyleMentor {
```

Comment:
I'm not 100% sure this will work as expected. Did you test that the styles are being added to the mentee modal as well?

Reply:
Yes, I added this line by line and it worked well.

Reply:
Perfect, good to know

Reply:
And I've removed unused styles